src/test/java/emissary/util/PayloadUtilTest.java

```diff
@@ -242,7 +243,7 @@ void testIsValidForm() {
         // Check that all expected valid characters are valid
         String alphaLow = "abcdefghijklmnopqrstuvwxyz";
         assertTrue(PayloadUtil.isValidForm(alphaLow), "Lower case alpha characters are expected to be valid");
-        assertTrue(PayloadUtil.isValidForm(alphaLow.toUpperCase()), "Upper case alpha characters are expected to be valid");
+        assertTrue(PayloadUtil.isValidForm(alphaLow.toUpperCase(Locale.ROOT)), "Upper case alpha characters are expected to be valid");
```

Comment:
Previously this would use `Locale.getDefault()`. Is there a general rule for these moving forward? It seems reasonable to me that any system/programmatic operations would benefit from the local insensitive `Locale.ROOT`. Since we know for sure this test case uses ascii characters, would this be a candidate for `Ascii.toUppperCase()`? In source code that deals with live payloads or metadata, would it be best to use `Locale.getDefault()`?